src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  width: 100%;
```

Comment:
It is not recommended to explicitly set the width of the body to 100%. The width of the body is already 100% by default.